### ISSUE TITLE
Code file name tag not accessible in dark mode

### ISSUE THREAD

**Issue by macdonst** [enhance-dev/enhance.dev#50]
When in dark mode the filename tab is not very readable and fails an a11y check. Should switch this color to something lighter when in dark mode.

![Screen Shot 2022-08-18 at 10 10 12](https://user-images.githubusercontent.com/353180/185416113-9b8bc46a-54ea-454c-830f-dfecda8df7ac.png)
